### DISCUSSION TITLE
NPM with base url

### DISCUSSION THREAD

**Discussion by lilp69** [NginxProxyManager/nginx-proxy-manager#3986] [Q&A]
Hi,
I just installed NPM and taked a ndd on ovh. Proxy works good for Grafana and NPM.
But I don't succeed to configure it for HomeAssistant, Zabbix and Nextcloud.
On OVH I make a DNS Zone like ha.ndd.tld, zabbix.ndd.tld, nextcloud.ndd.tld.
On NPM, I created a Proxy for each like :
HA : 
![image](https://github.com/user-attachments/assets/4ebb91ee-3632-4431-977d-f3e0b4ecfc7c)
Zabbix :
![image](https://github.com/user-attachments/assets/573574e8-d31e-4020-879b-4089ad775e6c)
![image](https://github.com/user-attachments/assets/883103a9-2b0b-4e63-8188-64c23de95fb9)
Nextcloud
![image](https://github.com/user-attachments/assets/cc5251b8-2125-4c71-8308-61c8e9f6fbef)
For zabbix and nextcloud, I have Apache default page.
For HA, I've 400 bad request.

I'm a little bit lost with thoses configuration.

**Comment by alex-1987** [comment]
I know only nextcloud use https
did you try to set to HTTPS? (Port 443)
or it use a other port?  Nextcloud use sometimes Port 8080

how du u connect to the webui?    use the same ports and address in the NPM

**Comment by lilp69** [comment]
With port 8080 I've "502 Bad Gateway" With 80 or 443, I've `ERR_TOO_MANY_REDIRECTS`
